lib/StoryEngine/DataLayer/model/FunctionalDescriptionDataModel.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace StoryEngine.StoryEngineDataModel;
+
+public record FunctionalDescriptionDataModel(Dictionary<string, int> ElementProminences)
```

Comment:
What is functional description again? 🤔

Reply:
A node's functional description identifies how it functions in the story in terms of developing theme, character, setting, and so on. It can also be used to tag nodes with information (for example, that a node takes place somewhere near water).

- prominence tags are used to indicate how prominently a story element appears in the scene; the values are used in algorithms determining which scenes to show a player next
- tag tags are used to add tag story elements to a node, which can be later used with prerequisites

(From the [XML format doc](https://docs.google.com/document/d/1Z2rmC1LnLx2QnttU8kyKBs8BWUJpL0ozEjcIfNYl4cc/edit?usp=sharing) from the prev Java version)